packages/superset-ui-core/test/callApi/callApiAndParseWithTimeout.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint promise/no-callback-in-promise: 'off' */
+import fetchMock from 'fetch-mock';
+
+import callApiAndParseWithTimeout from '../../src/callApi/callApiAndParseWithTimeout';
+
+// we import these via * so that we can spy on the 'default' property of the object
+import * as callApi from '../../src/callApi/callApi';
+import * as parseResponse from '../../src/callApi/parseResponse';
+import * as rejectAfterTimeout from '../../src/callApi/rejectAfterTimeout';
+
+import { LOGIN_GLOB } from '../fixtures/constants';
+import throwIfCalled from '../utils/throwIfCalled';
+
+describe('callApiAndParseWithTimeout', () => {
```

Comment:
nit
Add `()` in `describe('callApiAndParseWithTimeout()'`) so we know it is a function when reading test result